person_capture/gui_app.py

```
@@ -615,6 +615,22 @@ def _span_index_for(self, f, spans):
+
+    def _prescan_skip_forward(self, cap, count: int) -> int:
+        """Discard up to ``count`` decoded frames without restarting the reader."""
+        advanced = 0
+        remaining = max(0, int(count))
+        for _ in range(remaining):
+            if self._abort:
+                break
+            try:
+                if not cap.grab():
+                    break
+            except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)